refinery/tool_manager/serializers.py

```diff
@@ -68,6 +68,12 @@ class Meta:
 class ToolDefinitionSerializer(serializers.ModelSerializer):
     file_relationship = FileRelationshipSerializer()
     parameters = ParameterSerializer(many=True, allow_null=True)
+    workflow = serializers.SerializerMethodField()
+
+    def get_workflow(self, tool):
+        if tool.workflow:
+            return tool.workflow.uuid
+        return tool.workflow
```

Comment:
Wouldn't the 2nd `return tool.workflow` always be `None`here?

Reply:
Yes, am I being redundant?

Reply:
Maybe a little. How about something like
```
def get_workflow(self, tool):
    return tool.workflow.uuid if tool.is_workflow() else None
```
or
```
def get_workflow(self, tool):
    if tool.is_visualization():
        return None  # Vis Tools have no associated Workflows
    return tool.workflow.uuid
```

Reply:
I don't see how that improves anything.